### PR TITLE
[NP4861] Updated filter button and removed filter number

### PR DESCRIPTION
**Pull request by tala-abuadas**

### CODE REVIEW

File: src/foam/u2/filter/FilterView.js [hunk 1]
```diff
@@ -327,9 +329,6 @@ foam.CLASS({
             .start()
             .style({ overflow: 'hidden', 'align-self': 'center' })
             //TODO: remove when filter button gets a badge
```
Comment (sarthak-marwaha) [inline src/foam/u2/filter/FilterView.js]: remove comment

**Review by sarthak-marwaha — APPROVED** [review]
Just the one comment LGTM

File: src/foam/u2/filter/FilterView.js (hunk 1 above)
Comment (tala-abuadas) [reply src/foam/u2/filter/FilterView.js]: ok

File: src/foam/u2/filter/properties/PropertyFilterView.js [hunk 2]
```diff
@@ -227,12 +232,23 @@ foam.CLASS({
...
-        this.LABEL_PROPERTY_FILTER : this.LABEL_PROPERTY_ALL;
+        if ( this.view_.predicate !== this.TRUE && this.activeFilterCheck_ ) {
+          this.labelFiltering = this.LABEL_PROPERTY_FILTER;
+          this.filterController.activeFilterCount++;
+          this.activeFilterCheck_ = !this.activeFilterCheck_;
+        }
+        else if ( this.view_.predicate === this.TRUE && !this.activeFilterCheck_ ) {
+          this.labelFiltering = this.LABEL_PROPERTY_ALL;
+          this.filterController.activeFilterCount--;
+          this.activeFilterCheck_ = !this.activeFilterCheck_;
```
Comment (kgrgreer) [inline src/foam/u2/filter/properties/PropertyFilterView.js]: Space after !

File: src/foam/u2/filter/properties/PropertyFilterView.js [hunk 3]
```diff
@@ -227,12 +232,23 @@ foam.CLASS({
...
+        }
+
```
Comment (kgrgreer) [inline src/foam/u2/filter/properties/PropertyFilterView.js]: remove this line

**Pull request merged**